### PR TITLE
add date_from and date_to functionality for events

### PR DESCRIPTION
**Pull request by mattlub**
relates #107

- [x] update docs

I removed the functionality to filter by categories, as we hadn't fully decided what we wanted from it and it's not needed right now in nazareth-events.

### CODE REVIEW

**Comment by m4v15** [comment]
This breaks the categories filter though right?
@mattlub

**Comment by mattlub** [comment]
@m4v15 yep sorry, updated my initial comment.

**Review by m4v15 — APPROVED** [review]

File: tests/controllers/event.test.js [hunk 1]
```diff
@@ -38,15 +38,23 @@ tape('GET /events, with and without query parameters', t => {
           t.ok(eventNames.includes(validEvent3.en.name), 'third event has been added')
           t.deepEqual(eventStartTimes, eventStartTimes.sort(), 'returned events should be sorted by startTime')
         })
+      // supertest(server)
```
Comment (des-des) [inline tests/controllers/event.test.js]: commented code

File: docs.md [hunk 2]
```diff
@@ -38,6 +38,13 @@ Base URL: https://nazareth-open-tourism-platform.herokuapp.com/
 ### Get all events
 `GET /events`
 
+**Query parameters**
+
+Name | Type | Description
+--- | --- | ---
+date_from | date string (e.g. 2017-08-20) | All returned events should have a startTime later than or equal to 00:00 on this date, if supplied.
```
Comment (des-des) [inline docs.md]: The format of this string follows a spec: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

Either link this in or explain the options in more detail..

File: tests/controllers/event.test.js [hunk 3]
```diff
@@ -38,15 +38,23 @@ tape('GET /events, with and without query parameters', t => {
...
+        .get('/events?date_from=2017-05-01&date_to=2017-05-10')
         .expect(200)
         .expect('Content-Type', /json/)
         .end((err, res) => {
           if (err) t.fail(err)
-          // check our get path returns that event correctly
-          t.equal(res.body.length, 2, 'query response body should be an array with length 2')
-          t.ok(res.body.map(event => event.en.name).includes(validEvent2.en.name), 'results should be filtered correctly by url query parameters')
+          t.equal(res.body.length, 1, 'query response body should be an array with length 1')
+          t.ok(res.body.map(event => event.en.name).includes(validEvent2.en.name), 'results should be filtered correctly by url date-related query parameters')
```
Comment (des-des) [inline tests/controllers/event.test.js]: This line is too long

**Review by des-des — CHANGES_REQUESTED** [review]
wikid

**Review by des-des — APPROVED** [review]

**Pull request merged**